Cargo.toml

```diff
@@ -62,18 +37,15 @@ glob = "0.3"
 quickcheck = "0.6.2"
 
 [features]
-default = ["gif_codec", "jpeg", "ico", "png_codec", "pnm", "tga", "tiff", "webp", "bmp", "hdr", "dxt", "jpeg_rayon"]
+default = ["gif", "jpeg", "ico", "png", "pnm", "tga", "tiff", "webp", "bmp", "hdr", "dxt", "jpeg_rayon"]
 
-gif_codec = ["gif"]
-ico = ["bmp", "png_codec"]
-jpeg = ["jpeg-decoder"]
-png_codec = ["png"]
+ico = ["bmp", "png"]
 pnm = []
 tga = []
 webp = []
 bmp = []
 hdr = ["scoped_threadpool"]
 dxt = []
-jpeg_rayon = ["jpeg-decoder/rayon"]
+jpeg_rayon = ["jpeg/rayon"]
```

Comment:
Can we remove `jpeg` from the name of this feature, at least in the long term? I know it's a bit hypothetical but I'd want to have other decoders utilize SIMD and not only `jpeg`. Simply `rayon` might conflict with the package if we ever include it directly, do you have ideas for a better name?

Reply:
I don't know any better options

Reply:
It's fine for now, just would have been a nice-to-have.